waveshare2in13v2/doc.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
```

Comment:
The license format what slightly changed, causing the addlicense check to fail, please copy paste as is from another file, change the year to 2021.